CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## 0.5.1 - 2021-04-02
```

Comment:
As discussed offline, we have plenty of time to run an A/A test before our next actual release, so pick a date where no one (aka CTI) has to rush